library/src/main/java/com/google/android/exoplayer2/text/SubtitleDecoderFactory.java

```diff
@@ -76,7 +84,13 @@ public SubtitleDecoder createDecoder(Format format) {
         if (clazz == null) {
           throw new IllegalArgumentException("Attempted to create decoder for unsupported format");
         }
-        if (format.sampleMimeType.equals(MimeTypes.APPLICATION_CEA608)
+        if (format.sampleMimeType.equals(MimeTypes.TEXT_SSA)) {
+          byte[] header = format.initializationData.get(1);
+          String dlgfmt = new String(format.initializationData.get(0), "UTF-8");
```

Comment:
Please avoid abbreviations. I don't understand what "dlgfmt" means, and I doubt most other people will either ;).